packages/create-plugin/src/codemods/additions/additions.test.ts

```diff
@@ -0,0 +1,12 @@
+import defaultAdditions from './additions.js';
+
+describe('additions json', () => {
+  // as addition scripts are imported dynamically when add is run we assert the path is valid
+  defaultAdditions.forEach((addition) => {
+    it(`should have a valid addition script path for ${addition.name}`, () => {
+      expect(async () => {
+        await import(addition.scriptPath);
```

Comment:
Does this work reliably? When I bumped vitest to v4 I found that running vite locally would randomly cause these dynamic imports to throw errors as the workers would exit before the imported scripts imports had resolved. See comment in [migrations.test.ts](https://github.com/grafana/plugin-tools/pull/2233/files#diff-ac9b5cc72b04706837e33e52cdce7358b13d7bdba5b50e16dd881deba42ec5a0R6-R8).